docs/04-Bar-Chart.md

```diff
@@ -49,6 +49,7 @@ borderSkipped | `String or Array<String>` | Which edge to skip drawing the borde
 hoverBackgroundColor | `Color or Array<Color>` | Bar background color when hovered
 hoverBorderColor | `Color or Array<Color>` | Bar border color when hovered
 hoverBorderWidth | `Number or Array<Number>` | Border width of bar when hovered
+stack | `String` | For grouping stacked bars
```

Comment:
I think `String` is the best, because it serves as the name of the stack. For instance, here's chadcodes' image from the related issue 

![fff8fe0e-270e-11e6-8675-f2dfb0594676](https://cloud.githubusercontent.com/assets/300494/20095866/65c46794-a5fb-11e6-8dc1-95b12427bfca.png)

Also a number would imply an order (confusing if Dataset1 and 2 are in stack `2` and Dataset 3 is in stack `1`)